app/views/publishers/vacancies/vacancy_review_sections/_job_details.html.slim

```diff
@@ -17,7 +17,8 @@ h2 class="govuk-heading-m govuk-!-margin-bottom-4"
     - row.key
       = t("jobs.job_role")
     - row.value
-      = t("helpers.label.publishers_job_listing_job_role_form.job_role_options.#{vacancy.job_role}") if vacancy.job_role?
+      - if vacancy.job_roles.any?
```

Comment:
[Minor] I think you might be able to use the `readable_job_roles` method here

Reply:
Good catch, thanks for pointing it out!